mne/evoked.py

```diff
@@ -727,6 +727,42 @@ def plot_white(self, noise_cov, show=True):
         return _plot_evoked_white(self, noise_cov=noise_cov, scalings=None,
                                   rank=None, show=show)
 
+    def animate(self, ch_type, frames=5, interval=100, butterfly=False,
+                blit=True, show=True):
+        """Make animation of evoked data as topomap timeseries.
+
+        Parameters
+        ----------
+        ch_type : str
+            Channel type to plot. Accepted data types: 'mag', 'grad', 'eeg'.
+        frames : int | list of ints
+            If int, the number of frames to animate. If list of ints, the
+            indices to plot in the animation. Defaults to 5.
+        interval : int
+            The time interval before drawing a new frame as milliseconds.
+            Defaults to 100.
```

Comment:
You mean have the frame rate depend on the sfreq, so the animation appears
the same (in terms of data time to real time relationship, but not
smoothness) regardless of the decimation of the data?